### PR TITLE
remove duplicate command

### PR DESCRIPTION
**Pull request by ericsullivan**

### CODE REVIEW

File: lib/akd/base/init/distillery.ex [hunk 1]
```diff
@@ -91,7 +91,7 @@ defmodule Akd.Init.Distillery do
   # and forms a new command.
   # This currently supports only template
   defp rel_init(switches) when is_list(switches) do
-    Enum.reduce(switches, "#{setup()} \n mix release.init",
+    Enum.reduce(switches, "mix release.init",
```
Comment (aditya7iyengar) [inline lib/akd/base/init/distillery.ex]: Why are we removing this?

File: lib/akd/operation.ex [hunk 2]
```diff
@@ -71,11 +71,14 @@ defmodule Akd.Operation do
   def run(%__MODULE__{destination: %Destination{host: :local}} = operation) do
     Logger.info environmentalize_cmd(operation)
 
-    File.mkdir_p!(operation.destination.path)
+    path = operation.destination.path
+    |> Path.expand("/foo/bar/../bar")
```
Comment (aditya7iyengar) [inline lib/akd/operation.ex]: What is the `"/foo/bar/.../bar"` about?

File: lib/akd/base/init/distillery.ex (hunk 1 above)
Comment (ericsullivan) [reply lib/akd/base/init/distillery.ex]: Doesn't lin 82 already do it?

File: lib/akd/operation.ex (hunk 2 above)
Comment (ericsullivan) [reply lib/akd/operation.ex]: Copy paste error... Still worked though :)

**Comment by aditya7iyengar** [comment]
If we could update the tests that would be great!

**Pull request merged**